src/main/plugins/org.dita.html5/xsl/simpletable.xsl

```diff
@@ -29,7 +29,14 @@ See the accompanying LICENSE file for applicable license.
     <xsl:variable name="col-widths" as="xs:double*">
       <xsl:variable name="widths" select="tokenize(normalize-space(@relcolwidth), '\s+')" as="xs:string*"/>
       <xsl:for-each select="$widths">
-        <xsl:sequence select="xs:double(substring(., 1, string-length(.) - 1))"/>
+        <xsl:choose>
```

Comment:
Since the only allowed value space is `\d+(\.\d+)\*(\w+\d+(\.\d+)\*)*`, this could be refactored so that we do a `translate(@relcolwidth, '*', '')` in variable `$widths`, that way those values would already be doubles as strings. Then this part could just do casting to double.

Also, XPath 2.0 `. castable as xs:double` would IMO be more readable than the XPath 1.0 solutions to testing for NaN.